src/BookContent.h

```diff
@@ -0,0 +1,84 @@
+
+// BookContent.h
+
+#pragma once
+
+#include "WorldStorage/FastNBT.h"
+
+
+
+
+
+// tolua_begin
+class cBookContent
+{
+public:
+	/** Creates a empty book */
+	cBookContent():
+		m_IsSigned(false)
+	{}
+
+	/** Set the author of the book */
+	void SetAuthor(const AString & a_Author) { m_Author = a_Author; }
+
+	/** Returns the author of the book */
+	const AString & GetAuthor(void) const { return m_Author; }
+
+	/** Set the title of the book */
+	void SetTitle(const AString & a_Title) { m_Title = a_Title; }
+
+	/** Returns the title of the book */
+	const AString & GetTitle(void) const { return m_Title; }
+
+	/** Clears the whole book */
+	void Clear();
+
+	/** Returns true if the book has no author, no title and no pages */
+	bool IsEmpty(void) const;
+
+	// tolua_end
+
+	/** Required in ManualBindings to save the page as simple string or json string */
```

Comment:
I can't make any sense out of this comment, could you explain a bit more?

Reply:
Changed the comment. In a written book the content is a json string and in book and quill a simple string.